src/views/Wizard/views/__snapshots__/Mesh.spec.ts.snap

```diff
@@ -548,7 +548,7 @@ metrics:
                
               <p>
                 <a
-                  href="https://kuma.io/docs/null/policies/mesh/?utm_source=Kuma&utm_medium=Kuma-GUI"
```

Comment:
I know it's not a scope of this PR but there are other `nulls` in snapshots, like this - https://github.com/kumahq/kuma-gui/blob/1a9fe64e5591189e7f6ca17e10318d49ea670a47/src/views/__snapshots__/Shell.spec.ts.snap#L18 should I create an issue for it or is it ok in that place?

Reply:
I don’t think an issue is necessary. The underlying reason for these incorrect values is that a lot of the unit tests in this project take care of mocking its dependencies/data sources. Most of the tests instead rely on the mock data returned by the tools that mocks all (well, most) API responses. Since not all data always comes from APIs (e.g. some times, data is passed to components via props) and since not all APIs are mocked, this data is just sometimes missing. In this case, the tagline is `null` so this is the result. In any case, this isn’t indicative of an actual bug. It’s just something that could be nicer in the tests. That being said, I don’t think it has priority over other things that need to happen.